tests/conftest.py

```diff
@@ -1,16 +1,64 @@
 import pytest
 from pygqlc import GraphQLClient # main package
 
+class EnvironmentVariablesException(Exception):
+  """Check your environment variables"""
+
+class AuthorsModelException(Exception):
+  """There is a problem with the author model."""
+
+class RecordException(Exception):
+  """Record doesn't exists in author model"""
+
+def query_authors(name=""):
+  name_query = '{name:"'+f'{name}'+'"}'
+  if not name:
+    return "query {authors{id}}"
+  authors_query = "query {authors(filter:?){id}}".replace('?', name_query)
+  return authors_query
+
+
 @pytest.fixture(scope="session")
 def gql():
   import os
+  import sys
+
+  try:
+    if not os.environ.get('API'):
+      raise EnvironmentError
+    if not os.environ.get('WSS'):
+      raise EnvironmentError
+    if not os.environ.get('TOKEN'):
+      raise EnvironmentError
+  except EnvironmentError:
+    sys.exit("Check your environment variables")
+
   gql = GraphQLClient()
   gql.addEnvironment(
     'dev',
     url=os.environ.get('API'),
     wss=os.environ.get('WSS'),
     headers={'Authorization': os.environ.get('TOKEN')},
     default=True)
+
+
+  try:
+    _, errors = gql.query(query_authors(), flatten=True)
+    if not errors == []:
+      raise AuthorsModelException
+  except AuthorsModelException:
+    sys.exit(errors)
+
+  try:
+
+    _, errors_pau = gql.query(query_authors("Paulinna"))
+    _, errors_baruc = gql.query(query_authors("Baruc"))
+
+    if not (errors_baruc == [] and errors_pau == []):
```

Comment:
Una comparación de una lista con una lista vacía es repetitiva en python, las listas vacías son Falsey values.
Un par de opciones mas legibles son:
```suggestion
    if errors_baruc or errors_pau:
```

```suggestion
    if any([errors_baruc, errors_pau]):
```